### PR TITLE
add request timeout options

### LINKED ISSUE #6: Add timeout to requests
https://github.com/rokka-io/rokka.js/blob/master/src/transport.js#L10

Seems like it waits forever, if no timeout is specified

### PR DESCRIPTION
**Pull request by dwirz**
Closes #6

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const defaults = {
   renderHost: 'https://{organization}.rokka.io',
   apiVersion: 1,
   transport: {
+    requestTimeout: 10000,
```
Comment (chregu) [inline src/index.js]: Please set that timeout way higher. the request throtteling can easily let you wait for 20 seconds. So I'd say set this to 30 seconds

**Comment by dwirz** [comment]
Set request default timeout to 30s

File: src/index.js (hunk 1 above)
Comment (dwirz) [inline src/index.js]: 5d672cb

**Comment by chregu** [comment]
@pstadler: I merge that to help them with some issues they have, if you see a problem that we mix now promiseRetry and promiseRequest options, we can change it later, but it makes sense from the enduser point of view to have it there.

**Pull request merged**